kubech

```diff
@@ -26,6 +26,15 @@ EOF
 }
 
 
+#
+# Remember original kubeconfig.
+if [[ $KUBECONFIG ]]; then
+  kube_config_org=${KUBECONFIG}
```

Comment:
Please use 4 spaces since it's the default indentation in this file.

Reply:
Please also let's follow the conventions and call the var `kube_config_file_orig`.